src/codemagic/tools/_app_store_connect/action_groups/beta_groups_action_group.py

```diff
@@ -0,0 +1,82 @@
+from typing import Sequence
+from typing import Union
+
+from codemagic import cli
+from codemagic.apple import AppStoreConnectApiError
+from codemagic.apple.app_store_connect.builds import Builds
+from codemagic.apple.app_store_connect.provisioning.beta_groups import BetaGroups
+from codemagic.apple.resources import Build
+from codemagic.apple.resources import ResourceId
+from codemagic.cli import Colors
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BuildArgument
+from ..errors import AppStoreConnectError
+
+
+class BetaGroupsActionGroup(AbstractBaseAction):
+
+    @cli.action('add-build',
+                BuildArgument.BUILD_ID_RESOURCE_ID_REQUIRED,
+                BuildArgument.BETA_GROUP_NAMES_REQUIRED,
+                action_group=AppStoreConnectActionGroup.BETA_GROUPS)
+    def add_build(self, build_id: Union[ResourceId, Build], beta_group_names: Sequence[str]):
```

Comment:
On a second thought, it would be better if this method name (not the action name though) was more unique. `add_build` is not very unique and there could be other resources for which one can add builds and they you'd end up with two methods with the same name. 
```suggestion
    def add_build_to_beta_group(self, build_id: Union[ResourceId, Build], beta_group_names: Sequence[str]):
```
Same holds for the `remove_build` method.